Tests/Test-ADOPSConnection.Tests.ps1

```diff
@@ -0,0 +1,30 @@
+param(
+    $PSM1 = "$PSScriptRoot\..\Source\ADOPS.psm1"
+)
+
+BeforeAll {
+    Remove-Module ADOPS -Force -ErrorAction SilentlyContinue
+    Import-Module $PSM1 -Force
+}
+
+Describe 'Test-ADOPSConnection' {
+    Context 'Parameters' {
```

Comment:
If there are no parameters we can remove this block completely as it only clutters the test file.

Reply:
Removed!